src/components/NavBar/NavBar.js

```diff
@@ -17,11 +17,11 @@ class NavBar extends Component {
           <Navbar.Toggle />
         </Navbar.Header>
         <Navbar.Collapse>
-          <Nav>
-            <NavItem>
+          <ul className="nav navbar-nav">
+            <li role="presentation">
               <Link to="/dashboard">Dashboard</Link>
-            </NavItem>
-          </Nav>
+            </li>
+          </ul>
```

Comment:
@MartyTheeMartian Was this to solve the nested `<a>` tag warning?

Reply:
@elimisteve Yes, <NavItem> component comes with an `<a>` tag. I just inspected the component, and mimicked it's structure and replaced the `<a>` tag